CONP_DATS_fields.md

```diff
@@ -1,51 +1,49 @@
+|Derived From| ||The sources this dataset is derived from.|The sources this dataset is derived from.|
+|Derived From|extraProperties - derivedFrom|**REQUIRED IF** | the dataset is a derived dataset, the URL of the dataset it is derived from is **REQUIRED**.  The original dataset must also be included as a submodule in the derived dataset.| Name of the source dataset used to generate this dataset.|
+|Parent dataset id|extraProperties - parent_dataset_id|**REQUIRE IF**| the dataset is a derived dataset, the parent dataset id (as specified in _conp_dataset/.gitmodules_) is **REQUIRED**.|Identifier (DOI) of the source dataset used to generate this dataset.|
+|Primary Publications|primaryPublications|**RECOMMENDED**| The primary publication(s) associated with the dataset, usually describing how the dataset was produced.| The primary publication(s) associated with the dataset, usually describing how the dataset was produced.|
+|Title|Primary Publications -> Title||The name of the publication, usually one sentence or short description of the publication.|The name of the publication.|
+|Publication Venue|Primary Publications -> Publication Venue||The name of the publication venue where the document is published (if applicable).|The name of the publication venue where the document is published (if applicable).|
+|Authors|Primary Publications -> Authors||Authors of the publication.|Authors of the publication.|
+|Affiliations|Primary Publications -> Authors -> Affiliations||Author affiliations.|Authors affiliations.|
+|Date|Primary Publications -> Dates||Relevant dates for the publication. If you provide a date, it must come with a description of the date.|Relevant dates for the publication. If you provide a date, it must come with a description of the date (i.e.: first submission, final approval, date of publication, ...).|
+|Identifier|Primary Publications -> Identifier -> Identifier|**RECOMMENDED**| A code uniquely identifying an entity locally to a system or globally.|A code uniquely identifying the publication locally to a system or globally.|
+|Identifier Source|Primary Publications -> Identifier -> Identifier Source|**RECOMMENDED**| Primary identifier for the publication. Provide a *Document Object Identifier (DOI)* if you have one.|Primary identifier for the publication unique identifier. Provide a *Document Object Identifier (DOI)* if you have one. |
+|Dimensions|dimensions|**RECOMMENDED**| The different dimensions (granular components i.e.: handedness, weight, number of time event X append per day,...) making up a dataset. Providing dimensions give more details about the data types.|The different dimensions (granular components) making up a dataset. Providing dimensions give more details about the data types.|
+|Identifier|Identifier -> Identifier|**RECOMMENDED**| A code uniquely identifying the dataset locally to a system or globally.|A code uniquely identifying the dataset locally to a system or globally.|
```

Comment:
```suggestion
|Identifier|Primary Publications -> Identifier -> Identifier|**RECOMMENDED**| A code uniquely identifying an entity locally to a system or globally.|A code uniquely identifying the publication locally to a system or globally. Provide a *Document Object Identifier (DOI)* if you have one.|
```

Reply:
@cmadjar I changed "an entity" for "the dataset" to differentiate with "the publication" for the primary publication (line 36).

Could you confirm that I should use "entity" in both place?

Reply:
Good catch! Thanks!